hubspot/serializers.py

```diff
@@ -168,6 +173,8 @@ class Meta:
             "id",
             "name",
             "amount",
+            "stage",
+            "status",
```

Comment:
status is already here (below "purchaser"),  formerly was equal to what is now stage in this PR: `ORDER_STATUS_MAPPING[instance.status]` .